R/prepare_ae_specific.R

```diff
@@ -93,6 +67,31 @@ prepare_ae_specific <- function(meta,
   pop_group <- collect_adam_mapping(meta, population)$group
   obs_group <- collect_adam_mapping(meta, observation)$group
 
+  # Check if the grouping variable is missing
+  # pop_grp <- vapply(meta$population, "[[", FUN.VALUE = character(1), "group")
```

Comment:
Please don't have commented out **code**.

Reply:
> Please don't have commented out **code**.

I haved removed the code. Thanks for review!

Reply:
Hi @nanxstats , since @wangben718 already removed the comments, I directly merged this PR to the master, considering there are two teammates who need the package for slide preparation early this week...